src/Microsoft.TestPlatform.Extensions.TrxLogger/ObjectModel/TestElement.cs

```diff
@@ -23,14 +23,19 @@ internal abstract class TestElement : ITestElement, IXmlTestStore
         protected TestId id;
         protected string name;
         protected string owner;
+        protected string description;
+        protected string cssProjectStructure;
+        protected string cssIteration;
         protected string storage;
         protected string adapter;
         protected int priority;
         protected bool isRunnable;
         protected TestExecId executionId;
         protected TestExecId parentExecutionId;
         protected TestCategoryItemCollection testCategories;
+        protected TestPropertyItemCollection testProperties;
         protected TestListCategoryId catId;
+        private System.Collections.Generic.IList<string> workItemIds;
```

Comment:
Any reason why this alone is private? Can we make this protected as well?

Reply:
No reason other than there was no need for it being protected. For consistency though sure I can change it